docs/src/user/cli/info.rst

```diff
@@ -6,10 +6,16 @@ Here is an example of all the sections provided by the command.
 
 .. code-block:: console
 
-   orion info orion-tutorial
+   orion info --name orion-tutorial
```

Comment:
Unless this was changed, the command is `orion info <exp name>` directly.

Reply:
It was changed.